### PR TITLE
#931 Optimize dataization log

### PR DESCRIPTION
**Pull request by mximp**
#931

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256 @Graur please check

File: eo-runtime/src/test/resources/jul.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 handlers = org.slf4j.bridge.SLF4JBridgeHandler
-.level=FINEST
\ No newline at end of file
+.level=INFO
```
Comment (yegor256) [inline eo-runtime/src/test/resources/jul.properties:2]: @mximp so you are turning it off?

Comment (mximp) [reply eo-runtime/src/test/resources/jul.properties:2]: @yegor256 Yes. For the normal builds it's absolutely unnecessary.
It also speeds up the build two times.
It should be enabled for debugging purposes only.

Comment (yegor256) [reply eo-runtime/src/test/resources/jul.properties:2]: @mximp maybe you can add some explanation to README for those who want to see logs? Where to go and what to set up in order to see the logs.

Comment (mximp) [reply eo-runtime/src/test/resources/jul.properties:2]: That's a good idea.. Done.

**Comment by mximp** [comment]
@yegor256 please try to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**